### DISCUSSION TITLE
How to achieve non-gradient sliders on osu!lazer

### DISCUSSION THREAD

**Discussion by gowon11** [ppy/osu#30778] [Q&A]
Was watching some old footage of osu!lazer and i noticed that back then, black sliders (presumably sliders with 0,0,0 as their SliderTrackOverride value) were actually pitch pitch black as opposed the regular look of sliders from stable which has a gradient style. Is this look still achievable? Like some slider style in the skin.ini or anything of the sort?
![zen_hI6B0HTFmG](https://github.com/user-attachments/assets/d6cb0551-73c3-45cd-b990-5de5af8a38fe)

**Comment by darkralts** [comment]
@gowon11 did you find out? i want to do this

**Reply by n0Oo0Oo0b** [reply]
Looking at the rest of the [video](https://youtu.be/h3SkCJtSU3E), it's evident that that version of lazer is very incomplete (e.g. missing storyboard support) so the slider gradient was probably just another missing feature and the pitch black slider isn't intended to be the final version.

**Comment by tadatomix** [comment]
It's not a non-gradient slider btw. I think the way, older builds handeled skinning was like "stretch everything to Triangles skin and call it a day". So for rendering these sliders Triangles's slider body was used, with applied color from skin ini. And Triangles's slider body is like a gradient from white to transparent white. Since this youtuber was playing every map with 100% BG dim and YUGEN skin has a fully black slider color, no way why it's look like "non-gradient"

**Reply by tadatomix** [reply]
Maybe there will be 2 outcomes:
- Triangles's skin style will never be used, so these sliders might've been close to inaccessible 
- It will be available, just like it was in very old stable builds, where you had 4 slider styles, changeable through skin.ini value

**Comment by darkralts** [comment]
i think it would be nice to implement this as an extra slider style in skins
